examples/platform/linux/BUILD.gn

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+config("chip_examples_project_linux_platform_main_config") {
+  include_dirs = [ "." ]
+}
+
+source_set("chip_examples_project_linux_platform_main") {
```

Comment:
Please use shorter names. "examples", "platform", and "linux" are in the directory name and do not need to be repeated here.

Suggest "main".